src/Classes/Password.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Classes;
+
+
+class Password
+{
+
+    public function checkEmail($params)
+    {
+        $query = $query = new \App\Classes\Query();
+
+        return $query->CustomSQL("SELECT email FROM password_resets WHERE token = :token AND expired_token = :expired_token LIMIT 1", $params);
+    }
+
+    public function checkToken($params)
```

Comment:
Similar to the comment above, this could be renamed to be more clear of what the actual code is doing. Is any value in the `timed_expired_token` considered a pass/fail? If so, this could be changed to `isTokenExpired` and have it return a boolean